docs/detections/prebuilt-rules/downloadable-packages/0-16-2/prebuilt-rule-0-16-2-mounting-hidden-or-webdav-remote-shares.asciidoc

```diff
@@ -0,0 +1,83 @@
+[[prebuilt-rule-0-16-2-mounting-hidden-or-webdav-remote-shares]]
+=== Mounting Hidden or WebDav Remote Shares
+
+Identifies the use of net.exe to mount a WebDav or hidden remote share. This may indicate lateral movement or preparation for data exfiltration.
```

Comment:
```suggestion
Identifies the use of `net.exe` to mount a WebDav or hidden remote share. This may indicate lateral movement or preparation for data exfiltration.
```